app_legacy/Helpers/database/game.php

```diff
@@ -191,8 +191,16 @@ function getGameMetadata(
     return $numAchievements;
 }
 
-function getGameAlternatives(int $gameID): array
+function getGameAlternatives(int $gameID, ?int $sortBy = null): array
 {
+    $orderBy = match ($sortBy) {
+        11 => "ORDER BY HasAchievements ASC, gd.Title DESC",
+        2 => "ORDER BY gd.TotalTruePoints, gd.ID ASC ",
+        12 => "ORDER BY gd.TotalTruePoints DESC, gd.ID DESC ",
```

Comment:
Why wouldn't you want the secondary sort to be on title too?

Reply:
Recommend swapping 2 to be DESC as I would expect that clicking on Points would immediately show me the games with most points.

Reply:
Yeah I agree! It makes the meanings different than the achievement sorting but that doesn't really matter

Reply:
Now you need to make the arrow reflect the the sort order:
![image](https://user-images.githubusercontent.com/32680403/229294149-5670469f-446b-4ad1-ac54-6d4af11c2996.png)

Up arrow implies ascending sort, but values are descending.